frontend/src/components/navigation/FileSelector.tsx

```diff
@@ -47,6 +47,41 @@ interface FSItem {
 	content_type?: string;
 }
 
```

Comment:
Actually on a second thought; on this widget, can you read the URL query parameter to get the current datasetid instead? This way you can avoid changing the submitExtraction.tsx
There is a pattern usually: 
http://localhost:3000/files/66671d1e15f93923fbe8b110?dataset=6660b59f23f92e24d2f8bb40&folder=null&verNum=6669fdff1d5e4ec1df0257cf

Your "best friend" suggested:
```
function getDatasetId(url) {
    const urlObj = new URL(url);
    const datasetId = urlObj.searchParams.get("dataset");
    return datasetId;
}
```